src/builders/indexable-builder.php

```diff
@@ -299,6 +299,29 @@ protected function save_indexable( $indexable, $indexable_before = null ) {
 		return $indexable;
 	}
 
+	/**
+	 * Build and author indexable from an author id if it does not exist yet.
+	 *
+	 * @param int $author_id The author id.
+	 *
+	 * @return false|Indexable
```

Comment:
This needs to have a return description. This is especially important when the returned value changes based on the logic in the function, so developers know what return value to expect in which conditions 🙂 .
```suggestion
	 * @return Indexable|false The author indexable if it has been built, `false` if it could not be built.
```